docs/synapse/devopsguides/ingest_intro.rst

```diff
@@ -0,0 +1,641 @@
+Ingest Subsystem
```

Comment:
This should have a better name - may be easier to name parts if this is split into a few pages.

Also open to moving this to the user guides space?